app/controllers/my_account_controller.rb

```diff
@@ -5,6 +5,7 @@ def destroy
     @user =
       User.includes(
         logs: %i[log_entries log_shares],
+        quiz_participations: %i[quiz_question_answer_selections],
```

Comment:
(Basically an unrelated change to fix a Bullet complaint that was causing an exception when I tried to delete my account in order to test how that would work re: comments made.)